lib/mcb/commands/providers/touch.rb

```diff
@@ -0,0 +1,11 @@
+summary 'Update a provider so that it will be at the top of the apiv1 results'
+usage 'touch <provider_code>'
+param :provider_code, transform: ->(code) { code.upcase }
+
+run do |opts, args, _cmd|
+  MCB.init_rails(opts)
+
+  provider = Provider.find_by(provider_code: args[:provider_code])
```

Comment:
```suggestion
  provider = Provider.find_by!(provider_code: args[:provider_code])
```
other mcb use bang to fail fast.